UPGRADE.md

```diff
@@ -1,5 +1,18 @@
 # Upgrade to 2.15
 
+## Changing the way how reflection-based mapping drivers report fields, deprecated the "old" mode
+
+In ORM 3.0, a change will be made how the `AttributeDriver` reports field mappings.
```

Comment:
Isn't this sentence missing a word? (might just be me not knowing all the subtleties of the english language, in which case please let me know).
```suggestion
In ORM 3.0, a change will be made regarding how the `AttributeDriver` reports field mappings.
```